post1.html

```diff
@@ -0,0 +1,359 @@
+          </li>
+          <li>
+            <a target="_blank" href="https://drive.google.com/open?id=1mXrj5KdIYTfb01b_PCnoasAA8MBIpZEO" class="dropdown-item nestedmenu">Final Year</a>
+          </li>
+        </ul>
+      </li>
+      <li class="dropdown-item dropdown dropright">
+        <a class="dropdown-toggle" id="dropdown2-1" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Subject Notes</a>
+        <ul class="dropdown-menu" aria-labelledby="dropdown2-1">
+          <li>
+            <a target="_blank" href="https://drive.google.com/open?id=1UNN6vWwD3OghWKXXYk5jsrPC-esnzAtt" class="dropdown-item nestedmenu">1st Year</a>
+          </li>
+          <li>
+            <a target="_blank" href="https://drive.google.com/open?id=1lP76R5LvfFHVYH5yQlC2aZFOaIqu2yV9" class="dropdown-item nestedmenu">2nd Year</a>
+          </li>
+          <li>
+            <a target="_blank" href="https://drive.google.com/open?id=1UqkfxmiAw5hJgbcZFW_mWRGHYW8uVp-j" class="dropdown-item nestedmenu">3rd Year</a>
+          </li>
+          <!-- <li>
+          <a target="_blank" href="https://drive.google.com/open?id=1iOvrFP12trGkyyi4pre4e9wdoyPHpP8r" class="dropdown-item nestedmenu">Final Year</a>
+        </li> -->
+      </ul>
+    </li>
+    <li class="dropdown-item dropdown dropright">
+      <a class="dropdown-toggle" id="dropdown2-1" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Past Papers</a>
+      <ul class="dropdown-menu" aria-labelledby="dropdown2-1">
+        <li>
+          <a target="_blank" href="https://drive.google.com/open?id=1xhNMyx-A7Twg9XdsF6EZsmX-H8KaJoGq" class="dropdown-item nestedmenu">1st Year</a>
+        </li>
+        <li>
+          <a target="_blank" href="https://drive.google.com/open?id=1HOfMim4Fit6gRWh-8eEOXzNrHy097kTw" class="dropdown-item nestedmenu">2nd Year</a>
+        </li>
+        <li>
+          <a target="_blank" href="https://drive.google.com/open?id=1ZPqZOYGo-uRwsqgiX_VldQqgwFHGu7iU" class="dropdown-item nestedmenu">3rd Year</a>
+        </li>
+        <!-- <li>
+        <a target="_blank" href="https://drive.google.com/open?id=1jlKH6ffVdqdEjaagkHupaRkoqd38d6yB" class="dropdown-item nestedmenu">Final Year</a>
+      </li> -->
+    </ul>
+  </li>
+</ul>
+</li>
+</ul>
+</div>
+</nav>
+
+<!-- <img id="one" src="imgs\circle_red.png">
+<img id="two" src="imgs\semi_red.png">
+<img id="three" src="imgs\semi_another_red.png">
+<img id="four" src="imgs\circle_red.png"> -->
+<!-- w3-content defines a container for fixed size centered content,
+and is wrapped around the whole page content, except for the footer in this example -->
+<div class="w3-content" style="max-width:1400px">
+
+  <!-- Header -->
+  <header class="w3-container w3-center w3-padding-32">
+    <h1><b>SPEC BLOG</b></h1>
+    <p style="letter-spacing:1px;">Welcome to the blog of <span class="w3-tag">Electronics</span></p>
+  </header>
+

```

Comment:
No need to add this to any of the subpages. To get design ideas for subpages, look for the current WordPress page of blogs.